spec/feature/unauthenticated_guest_spec.rb

```diff
@@ -142,8 +140,8 @@
                   shared_bathroom: true,
                   user_id: 1)
     visit listings_path
-    find(:css, "#category_id_1[value='1']").set(true)
-    click_link_or_button "Filter by listing category"
+    find(:css, "#category_id").set(value="1")
```

Comment:
Useless assignment to variable - `value`.<br>Surrounding space missing for operator '='.